make_ghpages/make_pages.py

```diff
@@ -147,6 +148,11 @@ def get_aiida_version(setup_json):
             print("  >> AIIDA VERSION NOT SPECIFIED")
             return None
 
+        # precedence of version specs, from high to low
+        precedence = ['==', '>=', '>', '<=', '<']
+        sort_order = {precedence[i]: i for i in range(len(precedence))}
+        aiida_specs = sorted(aiida_specs, key=lambda r: sort_order[r[0]])
```

Comment:
Thanks! One more thing, though: use `sort_order.get(r[0], 100)` or some better way to avoid to get a key error for other operators (e.g. `a~=2.0` would return `~=` as an operator

Reply:
fixed!